README.md

```diff
@@ -19,16 +19,16 @@ A boilerplate repo.
 
 ### Iteration 0 - Main Page
 
-![screenshot of main page showing poster](/Iteration_0_Screenshot.jpg)
```

Comment:
I'm happy Leigh found a way to put the pictures in through a command in the Terminal.